x-pack/plugins/apm/common/service_runtime_name.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function isJRubyServiceRuntimeName(serviceName?: string) {
+  return serviceName?.toLowerCase() === 'jruby';
```

Comment:
Is this sent with different casing across versions since this is necessary? (I've seen it elsewhere and wondered the same there)

Reply:
Actually, I'm not sure, that's why I added `toLowerCase` before comparing it, just for safety.